app/services/pipeline_service/serializers/canvas_api_enrollment.rb

```diff
@@ -0,0 +1,55 @@
+module PipelineService
```

Comment:
This class looks a lot like the 'assignment' serializer.  I didn't swap out the enrollment serializer wholesale yet because I don't want to affect pipeline without one of their team members verifying that were doing the right thing as we go.